DataFormats/L1Trigger/src/HOTPDigiTwinMux.cc

```diff
@@ -0,0 +1,27 @@
+#include "DataFormats/L1Trigger/interface/HOTPDigiTwinMux.h"
+#include <cstdio>
+#include <iostream>
+
+using namespace std;
+
+HOTPDigiTwinMux::HOTPDigiTwinMux(int ieta, int iphi, int bx, int mip, int validbit, int wheel, int sector, int index, int link) {
+
+  if((ieta > 15) || (ieta < -15)) printf("HOTwinMuxDigi:  ieta out of range");
+  if ((mip < 0) || (mip > 1)) printf("HOTwinMuxDigi: mip value out of range");
+ 
+  theTP_HO=(abs(ieta)&0xF) | ((ieta<0)?(0x10) : (0x00)) |
```

Comment:
to avoid possible ambiguity coming into in the code in the future - please use std::abs explicitly even with using namespace std.